src/pages/standup/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { FC } from 'react';
+import Head from '@/components/head';
+import Layout from '@/components/Layout';
+
+import { LOGIN_URL } from '@/constants/url';
+import useAuthenticated from '@/hooks/useAuthenticated';
+import StandUpContainer from '@/components/standup';
+import { useGetUserQuery } from '@/app/services/userApi';
+import { skipToken } from '@reduxjs/toolkit/dist/query';
+
+const StandUp: FC = () => {
+    const { isLoggedIn, isLoading } = useAuthenticated();
+
+    const { isLoading: isAuthenticating } = useGetUserQuery(skipToken);
+
+    const handleConditionalRendering = () => {
+        if (!isAuthenticating && isLoggedIn) {
+            if (isLoading) {
+                return <p>Loading...</p>;
+            } else {
+                return <StandUpContainer />;
+            }
+        } else {
+            <div>
+                <p>You are not Authorized</p>
+                <a href={LOGIN_URL} target="_blank" rel="noreferrer">
+                    Click here to Login
+                </a>
+            </div>;
+        }
+    };
+
+    return (
+        <Layout>
```

Comment:
why are we using layout here ?
isn't this supposed to be wrapped in the _app.tsx

Reply:
The layout component is the parent component in which components like mine-task, standup, etc. are wrapped